osx_10.15.6.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script builds the repo version of PyNE (with the MOAB optional
+# dependency) from scratch on MacOS 10.15.6. The folder $HOME/opt is created
+# and PyNE is installed within.
+#
+# Run this script from any directory by issuing the command where <version>
+# is either "dev" or "stable":
+# $ ./osx_10.15.6.sh <version>
+# After the build finishes run:
+#  $ source ~/.bashrc
+# or open a new terminal.
+
+# Use package manager for as many packages as possible
+brew_package_list="glib python3 wget \
+             git cmake vim emacs gcc openblas \
+             lapack autoconf libtool make"
```

Comment:
this is missing `eigen`

Reply:
Thanks for the catch, is there anything else you noticed missing on the list? I tried to break up the ubuntu packages (like build-essential, python-software-properties, and software-properties-common), but some may have slipped through the cracks.

Reply:
hdf5 is also missing :)